crane-cloud/frontend
# static designs for project collaboration

# Description

This PR is to implement static designs for project collaboration.

## Type of change

Please delete options that are not relevant.



## Trello Ticket ID

https://trello.com/c/hdkwMobt

## How Can This Been Tested?

For membership, check out the projects settings page,
For pending project approval, you need to edit the conditional render in ProjectCard
 component for it to appear.
![image](https://user-images.githubusercontent.com/69305400/193895620-a7d334c9-3f21-4cad-8427-6240b70eaafa.png)


## Checklist:


# Screenshots

![image](https://user-images.githubusercontent.com/69305400/193895737-5c4e8445-2be5-46f6-a8ac-edb5c3ca18a6.png)
![image](https://user-images.githubusercontent.com/69305400/193896019-1ec7e701-8dbd-40da-965d-86b6358631eb.png)
![image](https://user-images.githubusercontent.com/69305400/193896125-5a139b9f-10d2-4b74-b8d8-153331cbdfdb.png)
![image](https://user-images.githubusercontent.com/69305400/193896438-03677aba-22b4-4e73-aa47-154dd22b07a4.png)
